static/css/style-v2.css

```
@@ -2621,4 +2621,380 @@ body {
+    gap: 12px;
+    padding: 10px 16px;
+    border-bottom: 1px solid var(--border);
+    transition: background 0.1s ease;
+}
+
+.env-var-row:hover {
+    background: var(--bg-hover);
+}
+
+.env-var-name {
+    font-family: "Cascadia Code", "Consolas", monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)